relayer/cmd/run/execution/command.go

```diff
@@ -64,6 +64,7 @@ func run(_ *cobra.Command, _ []string) error {
 
 	err = config.Validate()
 	if err != nil {
+		logrus.WithError(err).Fatal("config file validation failed")
```

Comment:
Doesnt fatal exit the program? Meaning that the `fmt.Errrof` will never return? I think we need to see where this error is bubbled up and put the Fatal there.